instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/coralogix/span/sqs.py

```diff
@@ -0,0 +1,103 @@
+"""
+This module provides functionality for creating OpenTelemetry spans for SQS events.
+"""
+
+from typing import List, Mapping, Optional, Tuple
+
+from opentelemetry.semconv.trace import SpanAttributes
+from opentelemetry.trace import set_span_in_context, Span, SpanKind, Link, span
+from opentelemetry.context.context import Context
+from opentelemetry.propagate import get_global_textmap
+from opentelemetry.trace.propagation import get_current_span
+from opentelemetry.propagators import textmap
+from opentelemetry.instrumentation.aws_lambda.utils import limit_string_size
+
+from opentelemetry.instrumentation.aws_lambda.coralogix.span import CoralogixSpan
+from opentelemetry.instrumentation.aws_lambda.coralogix import attributes as cx_attributes
+
+class SQSSpan(CoralogixSpan):
```

Comment:
I think this is missing the request body attribute, no?

Reply:
In the nodejs code we have the body for both SNS and SQS, that's why I asked. Weird that Python didn't have it for SQS. I guess we can add it later if that's the case, not required now.

Reply:
It is set. In the `messaging.message` attribute. I ported the thing from the other code. We can change.

Reply:
I did a second check and yes.. we are getting two "body" attributs and it seems that confused me. Thanks!
```python
span.set_attribute("messaging.message",
                                            limit_string_size(lambda_event["Records"][0].get("body")))
```